hosted_control_planes/hcp-deploy/hcp-deploy-ibmz.adoc

```diff
@@ -66,7 +66,19 @@ include::modules/hcp-ibmz-zvm-agents.adoc[leveloffset=+2]
 
 include::modules/hcp-ibmz-scale-np.adoc[leveloffset=+1]
 
```

Comment:
Remove line 69 from this assembly file.

Move lines 71 - 75 out of this file and into the `hcp-disconnected/disconnected-install-ibmz-hcp.adoc` assembly file.